src/Stripe.net/Services/Invoices/UpcomingInvoiceListLineItemsOptions.cs

```diff
@@ -42,8 +42,8 @@ public class UpcomingInvoiceListLineItemsOptions : ListOptions
         [JsonProperty("subscription_items")]
         public List<InvoiceSubscriptionItemOptions> SubscriptionItems { get; set; }
 
-        [JsonProperty("subscription_prorate")]
-        public bool? SubscriptionProrate { get; set; }
+        [JsonProperty("subscription_proration_behavior")]
+        public string SubscriptionProrationBehavior { get; set; }
```

Comment:
@topherdavis Could you keep the original one and add the new one instead? Otherwise this is a breaking change for developers that still use the old parameter and it would need a major version.

The best option is to do this:

```
        [Obsolete("Use SubscriptionProrationBehavior instead.")]
        [JsonProperty("subscription_prorate")]
        public bool? SubscriptionProrate { get; set; }

        [JsonProperty("subscription_proration_behavior")]
        public string SubscriptionProrationBehavior { get; set; }
```

Reply:
Thanks, makes sense. I wondered why SubscriptionProrate was still on `UpcomingInvoiceOptions`. Updated.